coopr-server/src/main/java/co/cask/coopr/spec/template/LeaseDuration.java

```diff
@@ -38,6 +40,50 @@ public LeaseDuration(long initial, long max, long step) {
     this.step = step;
   }
 
+  public LeaseDuration(String initial, String max, String step) {
+    this(getTimestamp(initial), getTimestamp(max), getTimestamp(step));
+  }
+
+  /**
+   * Returns a timestamp in milliseconds.
+   *
+   * @param base The string argument user provided.
+   * @return Timestamp in milliseconds
+   */
+  private static long getTimestamp(String base) {
+    if (base.equals("min") || base.equals("0")) {
```

Comment:
initial == 0 means there is no lease, and max == 0 means there is no max.  It is confusing, but I don't think we should change the behavior.  I think we should remove the "min" and "max" keywords.  Something like:
```
"leaseDuration": {
  "initial": "min",
  "max": "max",
  "step": "max"
}
```
would be very confusing for example.